### ISSUE TITLE
[Bug]: `sox` can't find `libmad.so.0` because it's actually called `libmad.so`

### ISSUE THREAD

**Issue by Rhys-T** [termux/termux-packages#18227]
### Problem description

I'm not sure exactly when this happened, but after updating packages at some point, I'm no longer able to run `sox`/`play`. All it does is output this error:
```
CANNOT LINK EXECUTABLE "sox": library "libmad.so.0" not found: needed by /data/data/com.termux/files/usr/lib/libsox.so.3.0.0 in namespace (default)
```
Upon further investigation, `libmad.so.0` is indeed missing from <code><var>$PREFIX</var>/lib</code>, and the actual library (_not_ a symlink) is installed under the base `libmad.so` name. Creating a symlink to it called `libmad.so.0` (i.e. in the opposite direction from how these symlinks normally go) gets `sox`/`play` working again.

I've tried `pkg reinstall`ing both `sox` and `libmad`[^libsoxr], but it still gets that error. Even if I move `usr` aside, let Termux create a new prefix, `pkg update` everything, and then `pkg install sox`, I still can't get it to run without creating that symlink.

[^libsoxr]: Also `libsoxr`, but I just realized that that's unrelated to `sox`/`libsox`.

(Unrelated weirdness that I just noticed while putting this together: Even with the symlink, `sox` doesn't seem to know what version it is - it just says `sox:      SoX v`, with no version number after the `v`. I can open this as a separate issue if you want.)

### What steps will reproduce the bug?

```console
$ pkg install sox
$ sox --version
CANNOT LINK EXECUTABLE "sox": library "libmad.so.0" not found: needed by /data/data/com.termux/files/usr/lib/libsox.so.3.0.0 in namespace (default)
```

### What is the expected behavior?

```console
$ pkg install sox
$ sox --version
sox:      SoX v
```

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=2278
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirror.csclub.uwaterloo.ca/termux/termux-main stable main
# x11-repo (sources.list.d/x11.list)
deb https://mirror.csclub.uwaterloo.ca/termux/termux-x11 x11 main
# tur-repo (sources.list.d/tur.list)
deb https://tur.kcubeterm.com tur-packages tur tur-on-device tur-continuous
Updatable packages:
firefox/x11 118.0.2 aarch64 [upgradable from: 118.0.1]
termux-tools version:
1.39.1
Android version:
13
Kernel build information:
Linux localhost 4.19.113-27114284 #1 SMP PREEMPT Fri Aug 11 10:06:51 +07 2023 aarch64 Android
Device manufacturer:
samsung
Device model:
SM-T970
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.widget versionCode:13
com.termux.x11 versionCode:14
com.termux.api versionCode:51
com.termux.styling versionCode:30
```

**Comment by licy183** [comment]
Thanks for your report! Should be fixed in `libmad 0.16.4-1`.

**Comment by Rhys-T** [comment]
Took a little while for the new version to show up, but that seems to have fixed it. Thanks!
